packages/test-utils/dist/vue-test-utils.js

```diff
@@ -11223,12 +11223,12 @@ function enableAutoDestroy(hook) {
   isEnabled = true;
 
   hook(function () {
-    wrapperInstances.forEach(function (wrapper) {
+    wrapperInstances.forEach((wrapper) => {
```

Comment:
can you revert this? we only update `dist` when we do a release

Reply:
Done, we've reverted the code change.